### DISCUSSION TITLE
golangci-lint throwing no export data error

### DISCUSSION THREAD

**Discussion by amokkara** [golangci/golangci-lint#3244] [Q&A] [answered]
golangci-lint run  throws below error in my go code.

```
~/aditya/podman-poc # ../bin/golangci-lint run --timeout 2m
WARN [runner] Can't run linter goanalysis_metalinter: buildir: failed to load package unshare: could not load export data: no export data for "github.com/containers/storage/pkg/unshare" 
ERRO Running error: 1 error occurred:
        * can't run linter goanalysis_metalinter: buildir: failed to load package unshare: could not load export data: no export data for "github.com/containers/storage/pkg/unshare"
```     
        
 I ran go mod tidy, go mod vendor. Still doesnt work, There is no issue in compiling the project as well.

**Comment by ldez** [comment, marked as answer]
Hello,

the problem is related to the compilation: it can be related to a cgo dependency.

Edit: it was related to build tags

**Reply by amokkara** [reply]
@ldez Hi, thanks for pointing out. 
if i add few args to the build command , my compilation works

**CGO_ENABLED=0** go build -o test -**tags "remote exclude_graphdriver_btrfs btrfs_noversion exclude_graphdriver_devicemapper containers_image_openpgp"**


how do i make this work for lint as well? just adding CGO_ENABLED=0 is not helping

**Reply by ldez** [reply]
did you define the build tags inside the golangci-lint configuration?

**Reply by amokkara** [reply]
> did you define the build tags inside the golangci-lint configuration?

ummm..No. not sure how to do it. Could you please point me in that direction. Thanks

**Reply by ldez** [reply]
https://golangci-lint.run/usage/configuration/#run-configuration

**Reply by amokkara** [reply]
This resolved it. Appreciate the help. Thanks
